### PR TITLE
New RD labcoat and CMO labcoat buff.

### PR DESCRIPTION
**Pull request by DrSnips**
RD has been pretty lacking in the style department so it's time he got some new threads. This gives him his own labcoat that has decent overall utility resistances (minor bomb as well as some radiation, and anomoly) as well as buffing the CMO labcoat to be slightly better than the underling's labcoats(+20 bio resist than the standard labcoat). Also allows switchtools to fit in labcoats.
all coat sides
Based on professor Membrane from invader zim.
:cl:
* rscadd: RD now has their own labcoat, it's longer and protects you more from science related hazards.
* tweak: CMO labcoat now protects you a bit more from biological hazards.
* tweak: CMO now starts without their CMO coat, visit your locker to get a bluer one.
* rscadd: Switchtools now fit in labcoats.

### CODE REVIEW

**Comment by DrSnips** [comment]
membrane
For those too lazy to google professor membrane.

File: code/modules/clothing/suits/labcoat.dm [hunk 1]
```diff
@@ -85,3 +86,10 @@
 	base_icon_state = "labcoat_onc"
 	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 60)
 	species_fit = list(VOX_SHAPED, GREY_SHAPED)
+
+/obj/item/clothing/suit/storage/labcoat/rd
+	name = "Research director's Labcoat"
```
Comment (jknpj) [inline code/modules/clothing/suits/labcoat.dm]: Needs either \improper or the caps removed. The later would be better, same for everything else in this file, thanks oldcoders.

**Comment by DrSnips** [comment]
Anything else for ya?

**Comment by BurntDevil** [comment]
nice

**Comment by DrSnips** [comment]
For anyone concerned that the radiation resist on the RD coat looking high, you still get fucked badly by rads like the radio transmitter and SME. Minor rads like radioactive geneticists/bodies are where this helps really.

File: code/modules/clothing/suits/labcoat.dm [hunk 2]
```diff
@@ -45,43 +45,51 @@
 	desc = "Bluer than the standard model."
 	base_icon_state = "labcoat_cmo"
 	item_state = "labcoat_cmo"
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 70, rad = 0)
 	species_fit = list(VOX_SHAPED, GREY_SHAPED)
 
 /obj/item/clothing/suit/storage/labcoat/mad
-	name = "The Mad's labcoat"
+	name = "the Mad's labcoat"
```
Comment (Intigracy) [inline code/modules/clothing/suits/labcoat.dm]: This one should stay, "The Mad" is a person and with your current change it'll read as the the Mad's labcoat.

File: code/modules/clothing/suits/labcoat.dm [hunk 3]
```diff
@@ -45,43 +45,51 @@
...
 	desc = "It makes you look capable of konking someone on the noggin and shooting them into space."
 	base_icon_state = "labgreen"
 	item_state = "labgreen"
 	species_fit = list(VOX_SHAPED)
 
 /obj/item/clothing/suit/storage/labcoat/genetics
-	name = "Geneticist Labcoat"
+	name = "geneticist Labcoat"
```
Comment (Intigracy) [inline code/modules/clothing/suits/labcoat.dm]: labcoat

File: code/game/objects/structures/crates_lockers/closets/secure/scientist.dm [hunk 4]
```diff
@@ -38,7 +38,7 @@
 	new /obj/item/clothing/head/bio_hood/scientist(src)
 	new /obj/item/clothing/under/rank/research_director(src)
 	new /obj/item/clothing/under/dress/dress_rd(src)
-	new /obj/item/clothing/suit/storage/labcoat(src)
+	new /obj/item/clothing/suit/storage/labcoat/rd(src)
```
Comment (Intigracy) [inline code/game/objects/structures/crates_lockers/closets/secure/scientist.dm:41]: CMOs spawn with a labcoat and have a spare in their locker, is that your plan here too or nah?

Comment (DrSnips) [reply code/game/objects/structures/crates_lockers/closets/secure/scientist.dm:41]: I intended to just have them suit up from their locker, but it slipped my mind about cmo getting one round start. I will swap their round start worn one with a genetics lab coat.
I was trying to stick with the trend that heads look for their good coat in their lockers.

Comment (Intigracy) [reply code/game/objects/structures/crates_lockers/closets/secure/scientist.dm:41]: Be sure to add that to the changelog.

Comment (DrSnips) [reply code/game/objects/structures/crates_lockers/closets/secure/scientist.dm:41]: I will.

**Comment by DrSnips** [comment]
CMO round start coat changed to genetics blue and in the change log.

**Comment by N3X15** [comment]
I just realized I merged a balance change early, fuck.

**Comment by DrSnips** [comment]
Coats merged, hitting the server kill switch right now because the imbalance they create to the whole game.

**Comment by SonixApache** [comment]
labcreep
why would you make CMOs not start with their coat?

**Pull request merged**